### ISSUE TITLE
Issue connecting to civitai to check for updates to installed models

### ISSUE THREAD

**Issue by carmaz54** [LykosAI/StabilityMatrix#840]
Since I installed Stability Matrix Version 2.12.0- pre.1+6afe2ef when I go to check for updates to my installed models in the model browser all I get is an error stating it cannot connect to Civitai. It was fine on the last 2.12.0 beta update.

**Comment by mtkgrafix** [comment]
Having same issue with 2.11.7. Model Browser kicks back this error message consistently. I can, however, update metadata without issue so it appears to be able to connect to some part of the Civitai API. No issues connecting with web browser - and I've forced a new IP at router (just in case I ended up with an IP with a reputation).
<img width="330" alt="Screenshot 2024-08-19 at 7 09 53 AM" src="https://github.com/user-attachments/assets/453c06cb-c3f3-4aa5-9e35-9f19e4c28ac4">
